cozy.example.yaml

```diff
@@ -80,6 +80,10 @@ hooks: ./scripts/hooks
 auto_updates:
   schedule: "@cron 0 0 0 * * *"
 
+# Registries used for applications and konnectors
+registries:
+  - https://registry.cozy.io/
```

Comment:
Do you think we should use this registry by default when no config file is used (for the cozy-app-dev docker image for example)?

Reply:
That would be more useful I think.

Reply:
done in 60de01bf78d3f50de7ada58d5a2915ce37e1ac35